screenshots.json

```diff
@@ -2,35 +2,712 @@
+                        "selector": ".module",
+                        "fileName": "ActivateClipboardPad"
+                    }, {
+                        "action": "clearDrawings"
+                    }, {
+                        "action": "click",
+                        "link": ".col-clipboard button.t3js-toggle-all-checkboxes"
+                    }, {
+                        "action": "wait",
+                        "timeout": "1"
+                    }, {
+                        "action": "click",
+                        "link": ".col-clipboard [aria-label=\"Transfer the selection of records to clipboard\"]"
+                    }, {
+                        "action": "waitForText",
+                        "text": "Remove item"
+                    }, {
+                        "action": "makeScreenshotOfElement",
+                        "selector": "#clipboard_form",
+                        "fileName": "ClipboardWithMultipleItems",
+                        "captionText": "Multiple items stored in the clipboard"
+                    }, {
+                        "action": "switchToMainFrame"
+                    }, {
+                        "action": "openPageTreePath",
+                        "path": ["Congratulations", "Shared Content"]
+                    }, {
+                        "action": "switchToContentFrame"
+                    }, {
+                        "action": "waitForText",
+                        "text": "Shared Content"
+                    }, {
+                        "action": "drawBox",
+                        "selector": "button[title=\"Paste in clipboard content\"]"
+                    }, {
+                        "action": "makeScreenshotOfElement",
+                        "selector": ".module",
+                        "fileName": "PasteClipboardContent",
+                        "captionText": "The \"Paste into\" icon at the top of the list"
+                    }, {
+                        "action": "click",
+                        "link": "button[title=\"Paste in clipboard content\"]"
+                    }, {
+                        "action": "waitForModalDialogInMainFrame"
+                    }, {
+                        "action": "resizeWindow",
+                        "width": 1024,
+                        "height": 480
+                    }, {
+                        "action": "makeScreenshotOfElement",
+                        "fileName": "ClipboardWarning",
+                        "captionText": "The dialog box for confirming the paste action"
+                    }, {
+                        "action": "clickButtonInModalDialog",
+                        "buttonLink": "OK"
+                    }, {
+                        "action": "resizeWindow",
+                        "width": 1232,
+                        "height": 950
+                    }, {
+                        "action": "makeScreenshotOfElement",
```

Comment:
Use
```
{'makeScreenshotOfWindow', ..}
```
instead of
```
{'makeScreenshotOfElement', ..}
```
if you omit parameter "selector", as without this parameter the action `makeScreenshotOfElement` takes a screenshot of the whole window which is better described with `makeScreenshotOfWindow`.

See https://github.com/TYPO3-Documentation/t3docs-screenshots/blob/main/packages/screenshots/Classes/Runner/Codeception/Support/Helper/Typo3Screenshots.php#L120.